boba_community/turing-monsters/contracts/WithRecover.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract WithRecover is Ownable {
+
+    /// @dev Recovers ERC20 tokens accidentally sent to the contract.
+    function recoverERC20(address tokenAddress, uint256 tokenAmount) external onlyOwner {
+        IERC20(tokenAddress).transfer(owner(), tokenAmount);
+    }
+
+    /// @dev Recovers ETH accidentally sent to the contract.
+    function recoverGas() external onlyOwner {
```

Comment:
does this compete with the withdraw() on NFTMonsters? if so, can we make it overrideable?

Reply:
indirectly it does yes, but since both methods can be executed by the same person this shouldn't make a difference. 
But yes that's something we should argue about, might also depend on how the project is organized.